d4s2_api_v2/api.py

```diff
@@ -158,6 +159,32 @@ def get_object(self):
         return self._ds_operation(DDSProjectTransfer.fetch_one, dds_util, dds_project_transfer_id)
 
 
+class DDSProjectPermissionsViewSet(DDSViewSet):
+    serializer_class = DDSProjectPermissionSerializer
+
+    def get_queryset(self):
+        dds_util = DDSUtil(self.request.user)
+        project_id = self.request.query_params.get('project')
+        user_id = self.request.query_params.get('user')
+        # a project id is required due to the underlying implementation of DukeDS API
```

Comment:
If this endpoint, when un-filtered, returned all of the user's projects+permissions, then I think a `project_id` query parameter makes sense (as a filter on that list). But since this requires that parameter - and it's a specific ID and not a search term, it doesn't seem like query parameters are the best fit.

Perhaps it would make more sense to list project permissions via a detail route under the single project endpoint (`/duke-ds-projects/:id/permissions/`), as DukeDS does?

Reply:
I was trying to make it fit in with the other ember models. I think I can use and adapter to make it work with this approach in ember.